frontend/components/modal/MissingTranslations.vue

```diff
@@ -1,33 +1,39 @@
 <script setup lang="ts">
 const { $axios, $missingTranslations } = useNuxtApp()
-const { missingTranslations: { missingTranslationsVisible } } = useModal()
+const { loading: { withLoading }, missingTranslations: { missingTranslationsVisible } } = useModal()
 const { t } = useI18n()
 
 const prefix = 'components/modal/MissingTranslations'
 const existing = useState<Map<string, string>>(`${prefix}.existing`, () => new Map())
 const languages = ['en', 'es', 'fr', 'de']
 const tt = (key: string) => t(`${prefix}.${key}`)
 
-onMounted(async () => {
-  for (const lang of languages) {
-    const langData = await $axios({
+const onOpen = async () => {
+  await withLoading(async () => {
+    await Promise.all(languages.map((lang) => $axios({
```

Comment:
nit: Looks like there's still some unnecessary async/await here. You can just return the promise directly, e.g.

Same elsewhere, like `pages/portfolios.vue`

```suggestion
  await withLoading(() => Promise.all(languages.map((lang) => $axios({
```

Reply:
Good catch!